crates/rattler_installs_packages/src/resolve/dependency_provider.rs

```diff
@@ -474,7 +474,7 @@ impl<'p> DependencyProvider<PypiVersionSet, PypiPackageName>
         }) else {
             panic!(
                 "could not find metadata for any sdist or wheel for {} {}. The following artifacts are available:\n{}",
```

Comment:
Should we mention something like "but none could be built"?

Reply:
Changed the wording, I've used extracted because `.whl` or not really built. WDYT?